api/src/main/java/org/openmrs/OrderGroup.java

```diff
@@ -0,0 +1,197 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs;
+
+import org.openmrs.api.APIException;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @since 1.12
+ * Transcational Data for the OrderSet {@link org.openmrs.OrderSet}
```

Comment:
Typo: spelling is "Transactional".

Actually I would rephrase this javadoc to say:

Contains a group of {@link org.openmrs.Order}s that are ordered together within a single encounter, often driven by an {@link org.openmrs.OrderSet}. Not all orders in an encounter need to be grouped this way, only those that have a specific connection to each other (e.g. several orders that together make up a treatment protocol for some diagnosis could be grouped).